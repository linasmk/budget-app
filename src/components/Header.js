/* ===== App Dependencies ===== */
import React from "react";
import { NavLink } from "react-router-dom";
/* ========== Components ========== */
import ExpensesSummary from "./ExpensesSummary";

/* ========== Code ========== */
const Header = () => (
  <header className="header">
    <section className="header__content">
      <h1 className="header__title">
        <NavLink className="header__title" to="/" exact={true}>
          budgedo
        </NavLink>
      </h1>
      <ExpensesSummary />
    </section>
  </header>
);
export default Header;
