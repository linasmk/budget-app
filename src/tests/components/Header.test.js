import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Header from "../../components/Header";
import { TestScheduler } from "jest";

test("Should render Header correctly", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
