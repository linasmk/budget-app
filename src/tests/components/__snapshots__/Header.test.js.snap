// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<header
  className="header"
>
  <section
    className="header__content"
  >
    <h1
      className="header__title"
    >
      <NavLink
        className="header__title"
        exact={true}
        to="/"
      >
        budgedo
      </NavLink>
    </h1>
    <Connect(ExpensesSummary) />
  </section>
</header>
`;
